### DISCUSSION TITLE
programmatically export dataviewjs tables to CSV

### DISCUSSION THREAD

**Discussion by lorenzotala** [metawops/obsidian-table-to-csv-export#14] [General]
Hi Metawops,

I am trying to use your plugin to export a dataviewjs table automatically once it is updated.

Is there a way to include a dataviewjs command at the end of my block to do it?

Best,

Lorenzo

**Comment by metawops** [comment]
To be honest: I have no idea if a command a plugin adds can be called from inside a dataviewjs block.
Maybe that's more of a question for the guy who made dataview? 😳 
I haven't used Obsidian actively for quite a while now and won't be able to work on my plugin in the foreseeable future ...
